backup-manager/README.md

```diff
@@ -0,0 +1,130 @@
+# Backup Manager
+
+An automated backup management system for the Minecraft server infrastructure. This Spring Boot application runs as a separate container and manages scheduled backups and cleanup.
+
+## Features
+
+- **Scheduled Backups**: Automatically runs the `backup.sh` script once a day (default: 2 AM)
+- **Size Management**: Monitors backup directory size and removes oldest backups when exceeding limit
```

Comment:
Issue (blocking): The size limit does not appear to be configurable at this time. Modifying the environment variable has no effect.